docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -55,20 +55,18 @@ components:
 ----
 ====
 
-This example has two containers, each defining a single endpoint. The endpoint is an accessible port that can be made accessible inside the workspace. Each endpoint has a name and port, which is the port on which the server running inside the container is listening. See the following attributes that you can set on the endpoint:
+This example has two containers, each defining a single endpoint. The endpoint is an accessible port. Each endpoint has a name and port. The server running inside the container is listening on this port. See the following attributes that you can set on the endpoint:
```

Comment:
```suggestion
This example has two containers, each defining a single endpoint. The endpoint is a port that can be made accessible inside the workspace. Each endpoint has a name and a port. The server running inside the container is listening on this port. See the following attributes that you can set on the endpoint:
```

Reply:
made the change, so resolving the comment.